src/components/carousel/index.stories.mdx

```diff
@@ -0,0 +1,405 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import Carousel from ".";
+import Image from "../image";
+import Button from "../button";
+import Icon from "../icon";
+
+export const SampleCard = ({ src }) => (
+	<div>
+		<h3 style={{ padding: "0 0.25rem" }}>
+			<Image src={src} />
+		</h3>
+	</div>
+);
+
+export const SampleNextArrow = ({ onClick }) => (
+	<Button
+		style={{
+			position: "absolute",
+			top: "50%",
+			right: "-25px",
+		}}
+		accessibilityLabel="Next"
+		onClick={onClick}
+	>
+		<Icon name="ArrowRight" width="24" height="24" />
```

Comment:
Width and Height are number props and should be set as `width={24} height={24}` - This is causing some console error currently